app/assets/javascripts/application.js

```diff
@@ -12,6 +12,9 @@
 // (https://github.com/sstephenson/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery.min
+//= require jquery-ui.min
+//= require angular.min
```

Comment:
at the bottom of this "files" list, they were added to vendor/assets/javascripts.